### PR TITLE
User model now mirrors the ERD

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status

READY
## Documentation

[ ] I have fully documented features in this PR. Please put an `X` in the box to confirm.
## Migrations

YES
## Description

A few sentences describing the overall goals of the pull request's commits.

The goal of this pull request was to update the user model to mirror the design of our ERD.
## Related PRs

List related PRs against other branches:

n/a
## Resolves Issue Number
#32
## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

You must still delete your current migration folder as well as database before testing this pull request.
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.

``` sh
git pull --prune
git checkout 32-update-user-model
rm -rf Migrations/
rm -rf bangazon.db
bangazon migration
bangazon run
```
## Impacted Areas in Application

List general components of the application that this PR will affect:
- This file solely impacts the User model

### CODE REVIEW

**Comment by garrett-vangilder** [comment]
@TeamCharles/team this PR is ready for review. Please note that the migration files are not included due to possible merge conflicts.

**Review by matthamil — APPROVED** [review]
Looks good

File: Models/User.cs [hunk 1]
```diff
@@ -24,7 +24,15 @@ public class User
...
+    [Required]
+    public string City {get; set;}
+
+    [Required]
+    public string State {get; set;}
+
+    [Required]
+    public int ZipCode {get; set;}
     
-    public ICollection<Product> Products;
```
Comment (MattKraatz) [inline Models/User.cs:28]: I think this needs to stay

**Review by MattKraatz — COMMENTED** [review]
One change

File: Models/User.cs [hunk 2]
```diff
@@ -34,5 +34,8 @@ public class User
     [Required]
     public int ZipCode {get; set;}
     
+    [Required]
```
Comment (matthamil) [inline Models/User.cs]: I don't think this needs to be `[Required]`. In Steve's original code, this is not required.

**Review by matthamil — CHANGES_REQUESTED** [review]
Remove [Required] over line 38 in Models/Users.cs

**Review by matthamil — APPROVED** [review]

**Comment by daynewright** [comment]
👍

**Pull request merged**